docs/rubrics/d34-draw.py

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""D3-4 draw — CANÔNICO (reproduz d34-sample.json). Auditor-verified 2026-08-15.
+Procedimento real: pool ordenado firing-desc/nome-asc; RNG fresco Random(3407) POR POOL;
+sample k por estrato (S1=2/2, S2=3/5, S3=3/10); saída ordenada por nome."""
+import json, random, os
+_mpath = os.path.join(os.path.dirname(__file__), 'skills-matrix.json')
+if not os.path.exists(_mpath):
+    raise SystemExit(f'ERRO: {_mpath} ausente — derive de docs/audits/audit-matrix-2026-08-15.md')
+m = json.load(open(_mpath, encoding='utf-8', errors='replace'))
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)